app/api/__init__.py

```diff
@@ -43,11 +43,12 @@
 api.route(UserDetail, 'user_detail', '/users/<int:id>', '/notifications/<int:notification_id>/user',
           '/event-invoices/<int:event_invoice_id>/user', '/users-events-roles/<int:users_events_role_id>/user',
           '/speakers/<int:speaker_id>/user', '/access-codes/<int:access_code_id>/user',
-          '/email-notifications/<int:email_notification_id>/user')
+          '/email-notifications/<int:email_notification_id>/user', '/discount-codes/<int:discount_code_id>/user')
```

Comment:
Should be called marketer here. `/discount-codes/<int:discount_code_id>/<relationship-name>`